src/makeInfinityAdder.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  let sum = 0;
+
+  const adder = (args) => {
```

Comment:
You don't have a lot of arguments here, just one. So better to use another name, for example
```suggestion
  const adder = (value) => {
```